### PR TITLE
Adds Ugandan Shilling currency and currency symbol.

### PR DESCRIPTION
**Pull request by davidofug**
Hi Team,

Added Ugandan shilling currency, after a proposal made by someone at CMS AFRICA SUMMIT 16.

First pull request made by Ugandan under help from @mattyza 

@mikejolley

### CODE REVIEW

File: includes/wc-core-functions.php [hunk 1]
```diff
@@ -389,6 +390,7 @@ function get_woocommerce_currency_symbol( $currency = '' ) {
 		'TRY' => '&#8378;',
 		'TWD' => '&#78;&#84;&#36;',
 		'UAH' => '&#8372;',
+		'UGX' => 'UGX',
```
Comment (claudiosmweb) [inline includes/wc-core-functions.php]: Seems like the symbol is `USh`.
Can yon confirm it for me?
https://en.wikipedia.org/wiki/Ugandan_shilling

Comment (wampamba) [inline includes/wc-core-functions.php]: Sure Ush is right but UGX is popularly used.

Comment (wampamba) [inline includes/wc-core-functions.php]: https://www.facebook.com/dwampamba/posts/1595968117394934?comment_id=1596231997368546&notif_t=feed_comment&notif_id=1459711042081842
Link of a survey made on facebook, is proof UGX is widely used than the USh.

**Comment by wampamba** [comment]
Can't see the currency in woocommerce!

*David Wampamba*

Web designer

www.davidwampamba.com

http://twitter.com/@dwampamba

http://facebook.com/dwampamba

http://plus.google.com/+davidwampamba

http://about.me/wampambadavid
<http://plus.google.com/+davidwampamba>

skype: dwampamba

Phone: +256.704.255401

“Inspire people to greatness!”

**Comment by shivapoudel** [comment]
@wampamba Will be available in WooCommerce 2.6 xD

**Comment by wampamba** [comment]
Great. xD

**Pull request merged**